index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
```

Comment:
You'll want to add the meta tag referenced in week one to all HTML pages on your site

Reply:
Yeah...still not on these pages. Make sure that meta tag is here.